tests/lib/filters/test_sandbox_filter.py

```diff
@@ -0,0 +1,92 @@
+import pytest
+import datetime
+import dataclasses
+
+from lib.log_processor import ProcessedLogEntry
+from lib.filters.sandbox_filter import sandbox_filter
+
+
+@pytest.fixture()
```

Comment:
Is it worth moving the fixtures for all these filters into a conftest and renaming them and importing them 🤷 genuine question... if you are bored

Reply:
https://stackoverflow.com/questions/73191533/using-conftest-py-vs-importing-fixtures-from-dedicate-modules 
It's a philosophical question... how bored are you?

Reply:
Not for me, thanks! 😆